### ISSUE TITLE
The batch is so obfuscated with Chinese it is so sus. 

### ISSUE THREAD

**Issue by Nikolaaa24** [iwannet/Projects#2]
### Describe the feature

Why the fuck would you obfuscate it so bad when it is not malware (unless it is)?

### Before submitting a suggestion

- [X] This feature doesn't already exist.

- [X] This wasn't already suggested. (I have searched suggestions on GitHub and read the FAQ)

**Comment by iwannet** [comment]
Yeah, it's a long story, it's actually because I was using this myself and I didn't want my teachers to find out. And then I actually forgot that it was still obfuscated.
Do you want me to put the source code online, or deobfuscate the script ?

**Comment by Nikolaaa24** [comment]
whatever you want just notify me when you do one of those, if you don't want to post the source code then just deobfuscate it

**Comment by SmilerRyan** [comment]
I can't comment on the original code (I got round the chiniese, but theres still too many substitutions for me to work though), but based on my runnings in a sandbox (and as expected) it just runs 4 commands on a loop.

I made a one-liner that kills the same 3 processes and loops itself, to re-create this.

[Kill_Lanschool.bat.txt](https://github.com/iwannet/Projects/files/13868055/Kill_Lanschool.bat.txt)

**Comment by iwannet** [comment]
Here is the source code. Yes the code is really easy but i made this so its easier to use by non-techy people.
`@echo off


msg * "LanschoolAir_Blocker started successfully. Credits to iwannet, Check my website: www.iwannet.cc" 

for /l %%i in (1,1,2147483647) do (
    

	
	taskkill /im LSAirClient.exe /f 
	taskkill /im LSAirClientUI.exe /f 
	taskkill /im student.exe /f
	timeout /t 1
	
)




 

`
